### PR TITLE
Pablo-R2-book-updates-(CU-8676z6fzk)

### PR DESCRIPTION
**Pull request by Martin-Kiss**

### CODE REVIEW

**Comment by KaiserKarel** [comment]
Task linked: [CU-8676z6fzk Pablo R2 book updates](https://app.clickup.com/t/8676z6fzk)

File: docs/docs/products/pablo-overview.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ provide a complete suite of features that together form a protocol that is great
 
 Pablo offers multiple AMM-types, enhanced customizability for LPs and traders, liquidity-bootstrapping pools, 
 cross-chain assets through our IBC and EVM bridges, and single sided staking.
-Upon launch, Balancer pools based on [Balancers weighted math] will be available,
+Upon launch, Balancer pools based on [BalancerFi weighted math] will be available,
```
Comment (0xBrainjar2) [inline docs/docs/products/pablo-overview.md:9]: didn't we say xy=k pools will be available on launch and not balancer pools.

File: docs/docs/products/pablo/launch-pools.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+- PICA/KSM
+
+:::note
+
+Pablo’s native token, PBLO, is set for launch in late Q1 2023.
+During this time, the entire protocol will be governed by PICA.
+After the launch of fNFTs on Picasso, governance will solely be controlled by xPICA holders until the launch of PBLO.
+:::
+
+Unlike other DEXs, Pablo does not rely on a highly inflationary reward token to incentivize users.
```
Comment (0xBrainjar2) [inline docs/docs/products/pablo/launch-pools.md:17]: don't say this

File: docs/docs/products/pablo/launch-pools.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+Instead, liquidity providers(LPs) directly benefit from the success and adoption of Pablo.
+LPs are provided LP-tokens relative to the funds they add to a liquidity pool on Pablo.
+A percentage of the total buy/sell order and swap fees are distributed to liquidity providers who bond their LP-tokens.
```
Comment (0xBrainjar2) [inline docs/docs/products/pablo/launch-pools.md:20]: who provide liquidity - no bonding involved

File: docs/docs/products/pablo/launch-pools.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+Then the accumulated fees are added back to the pool, effectively resulting in a pro-rata redistribution to LPs
+based on the LP-token share they redeem and the total value of the pool.
+If a user adds/removes liquidity from a pool, creating a high degree of liquidity imbalance,
+the standard 0.3% fee applies.
```
Comment (0xBrainjar2) [inline docs/docs/products/pablo/launch-pools.md:24]: fee is only for swappiing. There is no fee for adding/removing liquidity

**Review by 0xBrainjar2 — CHANGES_REQUESTED** [review]
Please resolve

File: docs/docs/products/pablo/launch-pools.md (hunk 4 above)
Comment (Martin-Kiss) [reply docs/docs/products/pablo/launch-pools.md:24]: According to the comments on the original doc, there is a specific case for add_liquidity for critical imbalance.
@PoisonPhang @benluelo can you confirm or correct this?

File: docs/docs/products/pablo-overview.md (hunk 1 above)
Comment (Martin-Kiss) [reply docs/docs/products/pablo-overview.md:9]: Will replace with the final suggestion from @vimukthi-git previously.

Comment (Martin-Kiss) [inline docs/docs/products/pablo-overview.md:9]:
```suggestion
Upon launch, Pablo will support dual asset Constant product(x * y = k) pools based on [BalancerFi weighted math]
```

File: docs/docs/products/pablo/launch-pools.md (hunk 2 above)
Comment (Martin-Kiss) [inline docs/docs/products/pablo/launch-pools.md:17]:
```suggestion
:::
```

File: docs/docs/products/pablo/launch-pools.md (hunk 3 above)
Comment (Martin-Kiss) [inline docs/docs/products/pablo/launch-pools.md:20]:
```suggestion
A percentage of the total buy/sell order and swap fees are distributed to liquidity providers.
```

File: docs/docs/products/pablo/launch-pools.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+Pablo’s native token, PBLO, is set for launch in late Q1 2023.
+:::
+Instead, liquidity providers(LPs) directly benefit from the success and adoption of Pablo.
+LPs are provided LP-tokens relative to the funds they add to a liquidity pool on Pablo.
+A percentage of the total buy/sell order and swap fees are distributed to liquidity providers.
+Then the accumulated fees are added back to the pool, effectively resulting in a pro-rata redistribution to LPs
+based on the LP-token share they redeem and the total value of the pool.
+If a user adds/removes liquidity from a pool, creating a high degree of liquidity imbalance,
+the standard 0.3% fee applies.
```
Comment (Martin-Kiss) [inline docs/docs/products/pablo/launch-pools.md:20]:
```suggestion
If a user provides liquidity in only one asset to a pool, they are charged the pool fee rate for creating an imbalance.
```

File: docs/docs/products/pablo/launch-pools.md (hunk 4 above)
Comment (Martin-Kiss) [reply docs/docs/products/pablo/launch-pools.md]: Edited with input from @benluelo. Thank you.

**Review by 0xBrainjar2 — APPROVED** [review]

**Review by KaiserKarel — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Pull request merged**